standard/expressions.md

```diff
@@ -6377,6 +6454,8 @@ assignment_operator
     ;
 ```
 
+<!-- [Placeholder] 12.21.1 -->
+<!-- [Placeholder] 12.21.1 End -->
```

Comment:
### [Placeholder] 12.21.1

#### First Alternative

A certain compiler produces a compile time error in the following situation.

---
&lt;!-- [ToDo] This restriction also applies to null-coalescing assignment, ??= , which is a
     C# 8 feature not yet spec’ed – once it is need to ensure this restriction applies.
-->
The *unary_expression* left operand of an assignment may not be a *null_forgiving_expression*.

---

(As before the [ToDo] comment is part of the suggestion.)

#### Second Alternative

---
Remove placeholder

---

Reply:
I don't have an opinion on this one yet - we need to talk about it.